src/tests/components/Footer.spec.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+// import { expect } from 'chai';
```

Comment:
Since you are using `jest` expect for your assertions, it will make your work need to remove the dead code and I think uninstalling `chai` too will be great for us or what do you think?